magmi/inc/magmi_auth.php

```diff
@@ -44,10 +44,31 @@ private function validatePass($hash,$pass){
         $hash = explode(":",$hash);
         $cecheck = md5($hash[1] . $pass);
         $eecheck = hash('sha256',$hash[1] . $pass);
-        $valid=($cecheck == $hash[0] || $eecheck== $hash[0]);
+	$eecheckArgo = $this->getArgonHash($pass, $hash[1]);
+        $valid=($cecheck == $hash[0] || $eecheck== $hash[0] || $eecheckArgo == $hash[0]);
 
         return $valid;
     }
+	
+    private function getArgonHash($data, $salt = ''){
```

Comment:
did you copy this code from Magento? if yes please add a comment iwth info where did you get it from